src/main/resources/templates/aboutMe.html

```diff
@@ -6,49 +6,67 @@
-                    na Oddziale Ogólnopsychiatrycznym Zamkniętym w Węgorzewie, gdzie zdobyłam doświadczenie w pracy z osobami chorymi
-                    psychicznie, pracy grupowej, a także prwadziłam zajęcia koordynowane przez <strong>Poradnię
-                    Uzależnień.</strong><br>
-                    Zarówno w Szpitalu, Zakładzie Opiekuńczo-Leczniczym jak i <strong>Powiatowym Domu Samopomocy</strong> udzielałam
-                    wsparcia indywidualnego i prowadziłam diagnostykę pacjentów.<br>
-                    W ostatnim czasie pracowałam w <strong>Szpitalu (SPZOZ) w Kraśniku</strong> na Oddziale Neurologii z Pododdziałem
-                    Udarowym, Oddziale Rehabilitacji oraz Oddziale Dziennym Psychiatrycznym prowadząc terapię
-                    indywidualną i grupową.
-                </p>
-                <p class="about-me">
-                    Kończąc <strong>dwustopniowy kurs mediatora</strong> małżeństw uzyskałam dodatkowe kwalifikacje i narzędzia do pracy z parami.
-                    Od kilku lat pracuję w prywatnym gabinecie z młodzieżą, osobami dorosłymi oraz prowadząc terapię par.<br>
-                    Jestem członkiem Polskiego Towarzystwa Psychiatrycznego.
-                    <strong>Pracuję pod stałą superwizją w celu stałego doskonalenia warsztatu pracy.</strong>
-                </p>
-                <p class="about-me">
-                    Prywatnie jestem żoną, mamą dwójki dzieci, miłośnikiem książek i tańca:)
+                    organizowałam liczne szkolenia i warsztaty z zakresu psychologii i doradztwa zawodowego. Kilka lat
+                    pracowałam na stanowisku psychologa w <b>Poradni Psychologiczno-pedagogicznej</b> prowadząc diagnostykę
+                    oraz terapię dzieci i młodzieży.<br>Przez trzy lata pracowałam z osobami dorosłymi w <b>Szpitalu
+                    Psychiatrycznym</b> na Oddziale Dziennym oraz na Oddziale Ogólnopsychiatrycznym Zamkniętym w Węgorzewie,
+                    gdzie zdobyłam doświadczenie w pracy z osobami chorymi psychicznie, pracy grupowej, a także prwadziłam
+                    zajęcia koordynowane przez <b>Poradnię Uzależnień.</b>
                 </p>
 
-
+                <p class="mt-20px">
+                    Zarówno w Szpitalu, Zakładzie
+                    Opiekuńczo-Leczniczym jak i <b>Powiatowym Domu Samopomocy</b> udzielałam wsparcia indywidualnego i
+                    prowadziłam diagnostykę pacjentów.<br>W ostatnim czasie pracowałam w <b>Szpitalu (SPZOZ) w Kraśniku</b>
+                    na Oddziale Neurologii z Pododdziałem Udarowym, Oddziale Rehabilitacji oraz Oddziale Dziennym
+                    Psychiatrycznym prowadząc terapię indywidualną i grupową.
+                </p>
             </div>
+            <div class="col-25p">
+                <div class="flip-box rounded cursor-pointer float-right">
+                    <div class="flip-box-inner rounded">
+                        <div class="flip-box-front">
```

Comment:
i write this tekst for your information. good name for images is important for position page for google. Attribute "alt" is important too. now i think, i will make rename file to "Ewa Górska Kraśnik Terapeuta.png" tomorrow

Reply:
i commented this line: `<img src="/images/Ewcia.jpg" alt="Ewa Górska - Kraśnik Terapeuta"/>`

Reply:
probably that because of that google robot can fiend easier during the searching